app/models/member.rb

```diff
@@ -3,6 +3,7 @@ class Member < ActiveRecord::Base
   belongs_to :organization
   has_one :account, as: :accountable
   has_many :movements, through: :account
+  has_many :events, dependent: :destroy
```

Comment:
Would you mind to add this kind of change to the tests also? Thanks :heart_eyes: 

e.g. https://github.com/coopdevs/timeoverflow/blob/develop/spec/models/member_spec.rb#L8

Reply:
👉 https://github.com/coopdevs/timeoverflow/pull/368/commits/7f6a0877268a9a23d37264c1cdfe6b51f30c2697